controller/bin/install_target

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash -ex
```

Comment:
Remove -x? I think that would lead to a bunch of unnecessary prints.

Reply:
done